### DISCUSSION TITLE
High battery usage

### DISCUSSION THREAD

**Discussion by YellowLe0** [termux/termux-app#3409] [Q&A] [answered]
II'm pretty new to this app and it works perfectly for the program I want to run, however is there any way I can stop the heavy battery use? I usually leave running in the background with only ONE session using wakelock...

Any suggestions? Any way to stop this/lower it atleast?
![Screenshot_20230519-055826_Device_care](https://github.com/termux/termux-app/assets/78830091/f4343ad8-42e3-4976-94f9-b8083cc6323b)

**Comment by twaik** [comment]
Application stays in background as long as there is at least one background session. There is no information about background activity on the screenshot, only about the time application stays in background.

**Reply by YellowLe0** [reply]
oh, but do you have any methods that could decrease the high battery usage? my phone doesn't show the exact details but time & % use

that screenshot was taken on an s9 incase you'd want to know

**Reply by twaik** [reply]
The only thing you can do is to shut down background termux sessions with "Exit" button in Termux's notification.

**Reply by YellowLe0** [reply]
what if i have a program that i need to run?

**Reply by twaik** [reply]
You have no choise. Or you have running program, or you have low battery usage.

**Reply by YellowLe0** [reply]
hm ok, thanks for responding 

its pretty strange how it uses so much battery though

**Comment by agnostic-apollo** [comment, marked as answer]
The reason for high battery usage is that you have enabled the wakelock, which means device won't sleep. Without wakelock and any power excessive programs running, termux would consume negligible battery, regardless of how many sessions are running.
